### PR TITLE
[RORO-4908][EL] - Welsh support

### PR DESCRIPTION
**Pull request by elabeca**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -34,14 +34,14 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
...
-features.welsh-language-support = false
+features.welsh-language-support = true
 
 accessibility-statement.service-path = "/gvms-service-availability"
 
 # Replace play.i18n.langs with the commented out line below when your service has been fully translated into Welsh
 # to enable Welsh translations for all content, including the standard headers and footers.
 # play.i18n.langs = ["en", "cy"]
-play.i18n.langs = ["en-GB"]
+play.i18n.langs = ["en", "cy"]
```
Comment (tipperba) [inline conf/application.conf:44]: This needs to stay as "en-GB" rather than "en" to override the browser's locale to GB to ensure formatting is correct.

Comment (elabeca) [reply conf/application.conf]: thanks @tipperba, I've updated. Good to know.

**Review by tipperba — APPROVED** [review]

File: app/uk/gov/hmrc/customsservicestatusfrontend/controllers/BaseFrontendController.scala [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/customsservicestatusfrontend/controllers/BaseFrontendController.scala:2]: minor: as we have just created this file, can we change this to 2024 ?

File: conf/application.conf [hunk 3]
```diff
@@ -34,14 +34,14 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
...
-features.welsh-language-support = false
+features.welsh-language-support = true
 
 accessibility-statement.service-path = "/gvms-service-availability"
 
 # Replace play.i18n.langs with the commented out line below when your service has been fully translated into Welsh
 # to enable Welsh translations for all content, including the standard headers and footers.
 # play.i18n.langs = ["en", "cy"]
-play.i18n.langs = ["en-GB"]
+play.i18n.langs = ["en-GB", "cy"]
```
Comment (sureshhmrc) [inline conf/application.conf:44]: minor: we can remove above commented line 43

**Review by sureshhmrc — DISMISSED** [review]

File: conf/application.conf (hunk 3 above)
Comment (elabeca) [reply conf/application.conf:43]: updated - thanks @sureshhmrc

File: app/uk/gov/hmrc/customsservicestatusfrontend/controllers/BaseFrontendController.scala (hunk 2 above)
Comment (elabeca) [reply app/uk/gov/hmrc/customsservicestatusfrontend/controllers/BaseFrontendController.scala]: updated - thanks @sureshhmrc

**Review by sureshhmrc — APPROVED** [review]

**Pull request merged**